doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -444,7 +444,7 @@ Here are examples:
 Enabling and disabling notations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: {| Enable | Disable } Notation {? {| @string | @qualid } } {? := @one_term } {? ( {+, @enable_notation_flag } ) } {? {| : @scope_name | : no scope } }
+.. cmd:: {| Enable | Disable } Notation {? {| @string | @qualid {* @ident__parm } } } {? := @one_term } {? ( {+, @enable_notation_flag } ) } {? {| : @scope_name | : no scope } }
```

Comment:
Would you add some text below explaining `@ident__parm`?

Reply:
I was under the assumption that these @ident__parm play the same role as in `Notation @qualid {* @ident__parm } := ...` but if it is not clear enough, I could add something like "where in `@qualid {* @ident__parm }`, `{* @ident__parm }` are the parameters of the abbreviation??

Reply:
Maybe say something like ```The :n:`{* @ident__parm }` are the parameters for an abbreviation.  See :cmd:`Notation (abbreviation)`.```

FWIW, the parameters are not well described in `Notation (abbreviation)`, either.  There should probably an example there.

Reply:
I added a word about parameters of abbreviations.

I also added a word on the replacing notation variables with `_` and added examples.

Regarding `Notation (abbreviation)` the existing example of `reflexive` is an example that shows a parameter.

(Note: rendering of the changes to the manual will have to doublecheck.)